src/Modules/Installer/Console/InstallCommand.php

```diff
@@ -42,7 +42,8 @@ protected function configure(): void
     {
         $this
             ->setDescription(
-                'Install fork from the console using the configuration in the fork-cms-installation-configuration.yaml file'
+                'Install fork from the console using the configuration in
+                the fork-cms-installation-configuration.yaml file'
```

Comment:
Moet je hier niet met concat werken? Anders gaat die newline toch in de output komen?